scripts/bootstrap-aio.yml

```diff
@@ -152,6 +151,32 @@
       when:
         - TARGET is defined
         - TARGET == "mnaio"
+    - name: Set ironic scenario specific user variables
+      set_fact:
+        scenario_user_variables_overrides_ironic:
+          tempest_plugins:
+            - name: ironic-tempest-plugin
+              package: ironic
+          tempest_service_available_ironic: True
+          tempest_test_sets: defcore ironic_api
+
+    - name: Add ironic scenario user vars to user_variables_overrides
+      set_fact:
+        user_variables_overrides: "{{ user_variables_overrides|combine(scenario_user_variables_overrides_ironic)}}"
+      when:
+        - SCENARIO is defined
+        - SCENARIO == "ironic-multihypervisor"
+
+    - name: Set specific user variables for all scenarios but ironic
+      set_fact:
+        scenario_user_variables_overrides_non_ironic:
+          nova_virt_type: qemu
+
+    - name: Add non-ironic scenario user vars to user_variables_overrides
+      set_fact:
+        user_variables_overrides: "{{ user_variables_overrides|combine(scenario_user_variables_overrides_non_ironic)}}"
+      when: (SCENARIO is defined and SCENARIO != "ironic-multihypervisor") or
+            (SCENARIO is undefined)
```

Comment:
Is this getting set correctly?  Currently we set `nova_virt_type=qemu` when we have a `JOB_NAME` env variable, meaning the job was instantiated through Jenkins.  This doesn't seem to match the current when.